lib/game_runner.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "board.rb"
+
+class GameRunner
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run(output)
+    output.puts empty_board
+    puts output.string
```

Comment:
You had previously commented on whether the board prints in the tests. Is the intention that it doesn't?

Reply:
Yes, the board should only print to the terminal when running the game for real. One way you could get around that is to "run" the game in a different file from where the class is defined.

Reply:
alright, I did that in the last two commits. I'm going to merge for now but happy to have feedback, I'm futzing with StringIO and stdin a bunch

Reply:
More specifically, I was thinking you could make a `game_runner.rb` in the root directory and all it does is require this file and run the game, passing in `$stdout`. You wouldn't test that file, at least as part of this exercise.

Reply:
but since there is a puts in `run`, wouldn't that print in my test whenever I test `run`?

Reply:
This puts should be removed. When running for real, the `output` above will be `$stdout`, so it should print the board to the terminal.

Reply:
ahha!